src/uct/ugni/base/ugni_types.h

```diff
@@ -17,13 +17,10 @@ typedef struct uct_ugni_device {
     char             type_name[UCT_UGNI_MAX_TYPE_NAME];  /**< Device type name */
     char             fname[UCT_DEVICE_NAME_MAX];/**< Device full name */
     uint32_t         device_id;                 /**< Device id */
-    int              device_index;              /**< Index of the device in the
```

Comment:
After the refactoring it was unused. Removed to reclaim a few bytes from the struct.